lib/src/notification/notification.component.scss

```diff
@@ -0,0 +1,288 @@
+@import "~@sebgroup/bootstrap/scss/variables";
```

Comment:
There is a lot of custom css in here, the point of having SEBbootstrap is to avoid that.
This https://sebgroup.github.io/bootstrap/toast-notifications contains what you need 
you will notice that the classnames start with **toast** please use them and remove the css

Reply:
This is a notification component Omar and solely based on what we already have in react-component. Although i reckoned toast component is a replacement for what we already have, i think we can create another component called toast, tailored based on the bootstrap toast.Notification component along with the loader component is one of the highly adopted component to my understanding and changing it right away might not be a good idea. Perhaps another component is the way to go.

Reply:
Yeah, Toast component is not a replacement for Notification, it's a different one, it even have a different use case.